packages/core/test/actorLogic.test.ts

```diff
@@ -1032,3 +1264,25 @@ describe('composable actor logic', () => {
     );
   });
 });
+
+interface Deferred<T> {
+  resolve(value: T): void;
+  reject(reason: any): void;
+  promise: Promise<T>;
+}
```

Comment:
couldn't this be replaced everywhere with `PromiseWithResolvers<T>`?

Reply:
oops, forgot to remove that. thanks for catching this